fortran/src/Makefile.am

```diff
@@ -22,7 +22,7 @@ include $(top_srcdir)/config/lt_vers.am
 # Include src directory in both Fortran and C flags (C compiler is used
 # for linking).
 AM_CPPFLAGS+=-I$(top_srcdir)/src
-AM_FCFLAGS+=-I$(top_srcdir)/src -I$(top_srcdir)/fortran/src
+AM_FCFLAGS+=-I$(top_srcdir)/src -I$(top_srcdir)/fortran/src $(FC_MODINC)$(fmoddir}
```

Comment:
There is also a fortran folder in the hl directory.

Reply:
Updated